src/vs/workbench/services/positronHistory/test/common/executionHistoryService.test.ts

```diff
@@ -100,8 +101,17 @@ class TestRuntimeSessionService implements IRuntimeSessionService {
 	private readonly _onWillStartSession = new Emitter<IRuntimeSessionWillStartEvent>();
 	readonly onWillStartSession = this._onWillStartSession.event;
 
+	// Add missing properties to fix interface implementation
```

Comment:
Nit: Do we need this comment and similar below?

Reply:
Nope!